#!/bin/sh /usr/share/dpatch/dpatch-run
## 50_pod_autoload.dpatch by Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add autoload.

@DPATCH@

--- a/elisp/emacs-goodies-el/pod-mode.el
+++ b/elisp/emacs-goodies-el/pod-mode.el
@@ -145,6 +145,7 @@
   )
 
 ;; main
+;;;###autoload
 (defun pod-mode ()
   "Major mode for editing POD files (Plain Old Documentation for Perl)."
   (interactive)
